contribute/documentation.md

```diff
@@ -21,18 +14,16 @@ If you encounter any misspellings, or violations to the style guide, please let
 
 On every page in the documentation there are two links:
```

Comment:
If this will not be part of the public-facing documentation, then I suggest adding a link to the Grafana documentation.